linux_os/guide/system/network/network-iptables/directory_permissions_etc_iptables/rule.yml

```diff
@@ -2,7 +2,7 @@ documentation_complete: true
 
 title: Verify Permissions On /etc/iptables Directory
 
-description: '{{{ describe_file_permissions(file="/etc/iptables", perms="0600") }}}'
+description: '{{{ describe_file_permissions(file="/etc/iptables", perms="0700") }}}'
```

Comment:
Execution permission for this folder makes sense. However, I didn't find this folder specified in any package. Instead I found `/etc/sysconfig/iptables` provided by `iptables-service` package. Do you know the context where this folder is used or which policy currently mentions it? Perhaps it could be even be deprecated.

Reply:
At least RHEL I can't find it.  It appears the SLE also use this rule as well.

Reply:
Ok. In any case, the update is good. Thanks